commands.c

```diff
@@ -91,14 +91,20 @@ void commands_process_packet(unsigned char *data, unsigned int len,
 	switch (packet_id) {
 	case COMM_FW_VERSION: {
 		int32_t ind = 0;
-		uint8_t send_buffer[50];
+		uint8_t send_buffer[24 + HW_NAME_MAX_LENGTH];
```

Comment:
Here I would prefer to have a buffer with more than enough length rather than counting the bytes manually, as it is easy to get that wrong. Best would of course be to have an automatic way to determine the size, but I don't know of any nice way to achieve that in C without adding a lot of extra complexity.